setup.py

```diff
@@ -91,6 +91,7 @@ def get_cryptography_version():
         "python-multipart",
         "typing_extensions",
         "backports.cached_property",
+        "fastapi-jwt-auth",
```

Comment:
@giffels Should we make the REST API an optional `extra` perhaps? There is quite a bit of stuff pulled in for it now, not sure if everyone wants that much.

Reply:
+1 for making the REST API an optional `extra`.